src/services/apps-metadata.ts

```diff
@@ -4,7 +4,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
+import { PredefinedProperties } from '../utils/types';
 
+// https://github.com/gridsuite/deployment/blob/main/docker-compose/docker-compose.base.yml
+// https://github.com/gridsuite/deployment/blob/main/k8s/resources/common/config/apps-metadata.json
+export type MetadataJson = MetadataCommon | MetadataStudy;
```

Comment:
Why `MetadataJson` and not just `Metadata` ? Every object in JS is a JSON anyway.

Reply:
Fine by me

Reply:
because that is a file that we fetch, not an API of a server

Reply:
Not a good reason for me, we should not write "Json" at the end of a type because everything is a Json, whatever it come from, but maybe ask a TL to be sure

Reply:
If Metadata is not good, which I agree a bit, other possibility : 

- AppsMetadata
- CommonMetadata
- StudyMetadata
- etc...

For me it's even better.
@TheMaskedTurtle and the others, what do you think ?